lib/index.js

```diff
@@ -11,5 +12,6 @@ var main = function (options, callback) {
 
 module.exports = main;
 module.exports.launch = main;
+module.exports.argv = argv;
```

Comment:
Seems like is now really coupling it, maybe add another method which takes `process.argv` and parses it?

Like `sauce.launchWithArgv(process.argv)`?